examples/ccn-lite-relay/main.c

```diff
@@ -33,17 +29,13 @@
 #define MAIN_QUEUE_SIZE     (8)
 static msg_t _main_msg_queue[MAIN_QUEUE_SIZE];
 
-#ifndef BOARD_NATIVE
-    /* some buffer for the heap */
-#   define TLSF_BUFFER     (10240 / sizeof(uint32_t))
-    static uint32_t _tlsf_heap[TLSF_BUFFER];
-#endif
+/* some buffer for the heap */
+#define TLSF_BUFFER     (10240 / sizeof(uint32_t))
```

Comment:
Hm, not really, just tried to find a sufficient big number that would work on all supported platforms and though 10kB would be a nice number.